README.md

```diff
@@ -1,2 +1,4 @@
 # python3-minidsp
 MiniDSP 2x4HD and DDRC24 driver re-written in Python
+
```

Comment:
Please remove fork notice (though if you want to add to the README that would be great!)